src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,15 @@
+package lotto.view;
+
+import static camp.nextstep.edu.missionutils.Console.readLine;
+import static lotto.util.Validator.*;
```

Comment:
Google Java Style Guide의 일부분입니다.

> 3.3 Import statements
3.3.1 No wildcard imports
Wildcard imports, static or otherwise, are not used.

wildcard import 하지 않는 법 : https://www.baeldung.com/intellij-disable-wildcard-import

Reply:
저도 이거 알고는 있었는데 자세히 안봤네요. 감사합니다.